ztp/extra-manifests-builder/05-kdump-config/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+MCPROLE=${MCPROLE:-master}
+
+# kdump-config MachineConfig removes the ice module from kdump to prevent kdump failures on
+# certain servers. This is a temporary workaround for RHELPLAN-138236 and can be removed when
+# that issue is fixed.
+
+kdump_remove_ice_module_base64=$(base64 -w0 kdump-remove-ice-module.sh)
+
+echo "\
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: ${MCPROLE}
+  name: 05-kdump-config-${MCPROLE}
```

Comment:
Would we want a more specific name like `05-kdump-config-icelake-fix`?

Probably not that important, and I'm fine with it as-is.

Reply:
I was thinking that it is possible there would be other issues in this area in the future where we might have to blacklist another driver (for example), so I wanted to use a more generic name for the machineconfig to allow for that.